src/CarbonAware/src/CarbonAwareConstants.cs

```diff
@@ -5,7 +5,8 @@
 /// </summary>
 public class CarbonAwareConstants
 {
-    public const string Locations = "locations";
```

Comment:
Adding a new field for single location and renaming constants for clarity (single vs. multiple)